src/utils/middleware/csrf.middleware.js

```diff
@@ -0,0 +1,79 @@
+const csrf = require('csrf')
+
+/**
+ * @param logger {Logger}
+ * @param sessionKey {string}
+ * @param secretKey {string}
+ * @param tokenKey {string}
+ */
+const configureCsrfMiddleware = (logger, sessionKey, secretKey = 'csrfSecret', tokenKey = 'csrfToken') => {
+  logger.debug('--- CSRF middleware configuration ---')
+  logger.debug(`Secret is set at req.[${sessionKey}][${secretKey}]`)
+  logger.debug(`Token is checked at req.body|query[${tokenKey}]`)
```

Comment:
extremely minor point but these log lines don't strictly parse properly as JS, it should either be `req.${sessionKey}.${secretKey}` or `req['{sessionKey}']['${secretKey}']` and similarly for the second line